src/morphometrics/measure/label.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+from skimage.measure import regionprops_table
+
+from ..types import LabelImage, LabelMeasurementTable
+from . import register_measurement
+
+
+@register_measurement(name="volume")
+def volume(label_im: LabelImage) -> LabelMeasurementTable:
+    rp_table = regionprops_table(label_im, properties=("label", "area"))
+
+    return pd.DataFrame(rp_table).set_index("label")
+
+
+@register_measurement(name="centroid")
+def centroid(label_im: LabelImage) -> LabelMeasurementTable:
+    rp_table = regionprops_table(label_im, properties=("label", "centroid"))
```

Comment:
That's a tricky pattern. If I'm not wrong, this will have sub-optimal performance, especially on big images. We might want to explore a pattern where we can cache the `regionprops` object.

Reply:
Let me just add a link. I used an alternative pattern, that could be a starting point. Imagine there is an object that holds `regionprops`, in case any function in a pool of functions requested it. If a second functions asks for `regionprops`, it would access the same object as the first function used before. Not sure if the term is right, but that's some kind of "singleton" on `regionprops`-object level. You can read an implementation of this [here](https://github.com/clij/clijx-accelerated-pixel-and-object-classification/blob/main/src/main/java/net/haesleinhuepf/clijx/apoc/LabelFeatureGenerator.java#L221) (sorry, in Java)

Reply:
Agreed. As discussed below, I've introduced the concept of "measurement sets" with selectable measurements to allow groups of measurements be easily computed with cacheing.